presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -121,6 +121,24 @@ public void testCreateTable()
     }
 
     @Test
+    public void testCreateTableLike()
```

Comment:
This will need to be overridden in every connector that doesn't support table creation. Instead, keep this as part of `testCreateTable()`.